scripts/openmc-get-photon-data

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+"""
+Download ENDF/B-VII.1 ENDF data from NNDC for photo-atomic and atomic
+relaxation data and convert it to an HDF5 library for use with OpenMC.
+This data is used for photon transport in OpenMC.
+"""
+
+import os
+import sys
+import shutil
+import zipfile
+import argparse
+from io import BytesIO
+from urllib.request import urlopen
+
+import openmc.data
+
+
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
+                      argparse.RawDescriptionHelpFormatter):
+    pass
+
+parser = argparse.ArgumentParser(
+    description=__doc__,
+    formatter_class=CustomFormatter
+)
+parser.add_argument('-c', '--cross-sections',
+                    help='cross_sections.xml file to append libraries to')
+args = parser.parse_args()
+
+base_url = 'http://www.nndc.bnl.gov/endf/b7.1/zips/'
+files = ['ENDF-B-VII.1-photoat.zip', 'ENDF-B-VII.1-atomic_relax.zip']
+block_size = 16384
+
+# ==============================================================================
+# DOWNLOAD FILES FROM NNDC SITE
+
+if not os.path.exists('photon_hdf5'):
+    os.mkdir('photon_hdf5')
+
+for f in files:
+    # Establish connection to URL
+    url = base_url + f
+    req = urlopen(url)
+
+    # Get file size from header
+    file_size = req.length
+    downloaded = 0
+
+    # Check if file already downloaded
+    if os.path.exists(f):
+        if os.path.getsize(f) == file_size:
+            print('Skipping ' + f)
+            continue
+        else:
+            overwrite = input('Overwrite {}? ([y]/n) '.format(f))
+            if overwrite.lower().startswith('n'):
+                continue
+
+    # Copy file to disk
+    print('Downloading {}... '.format(f), end='')
+    with open(f, 'wb') as fh:
+        while True:
+            chunk = req.read(block_size)
+            if not chunk: break
```

Comment:
I believe multiple statements on one line with a colon breaks PEP8.  See rule E701 [1][2]

Should be:
```
if not chunk:
  break
```

[1] https://lintlyci.github.io/Flake8Rules/rules/E701.html
[2] https://www.python.org/dev/peps/pep-0008/#other-recommendations